plugins/nodes/src/js/utils/compareValues.tsx

```diff
@@ -0,0 +1,21 @@
+// TODO: decide where this should live (taken from TableUtil)
```

Comment:
Where is the Jira where we will discuss where to put this?

Reply:
forgot it 😅